src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.operation.BalanceOperation;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.PurchaseOperation;
+import core.basesyntax.operation.ReturnOperation;
+import core.basesyntax.operation.SupplyOperation;
+import core.basesyntax.service.DataParser;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.serviceimpl.DataParserImpl;
+import core.basesyntax.serviceimpl.FileReaderImpl;
+import core.basesyntax.serviceimpl.FileWriterImpl;
+import core.basesyntax.serviceimpl.ReportGeneratorImpl;
+import core.basesyntax.serviceimpl.ShopServiceImpl;
+import core.basesyntax.strategy.Strategy;
+import core.basesyntax.strategy.StrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String FILE_FROM = "src/main/resources/input.csv";
+    private static final String FILE_TO = "src/main/resources/output.csv";

```

Comment:
Avoid using hardcoded file paths. Instead, pass the file paths as parameters to the main method or read them from a configuration file or environment variable.